src/pages/ChatFinderPage/index.tsx

```diff
@@ -25,33 +25,34 @@ import CONST from '@src/CONST';
 import ONYXKEYS from '@src/ONYXKEYS';
 import type SCREENS from '@src/SCREENS';
 import type * as OnyxTypes from '@src/types/onyx';
-import SearchPageFooter from './SearchPageFooter';
+import ChatFinderPageFooter from './ChatFinderPageFooter';
 
-type SearchPageOnyxProps = {
+type ChatFinderPageOnyxProps = {
     /** Beta features list */
     betas: OnyxEntry<OnyxTypes.Beta[]>;
 
     /** Whether or not we are searching for reports on the server */
     isSearchingForReports: OnyxEntry<boolean>;
 };
 
-type SearchPageProps = SearchPageOnyxProps & StackScreenProps<RootStackParamList, typeof SCREENS.SEARCH_ROOT>;
+type ChatFinderPageProps = ChatFinderPageOnyxProps & StackScreenProps<RootStackParamList, typeof SCREENS.CHAT_FINDER_ROOT>;
 
-type SearchPageSectionItem = {
+type ChatFinderPageSectionItem = {
     data: OptionData[];
     shouldShow: boolean;
 };
 
-type SearchPageSectionList = SearchPageSectionItem[];
+type ChatFinderPageSectionList = ChatFinderPageSectionItem[];
 
 const setPerformanceTimersEnd = () => {
-    Timing.end(CONST.TIMING.SEARCH_RENDER);
-    Performance.markEnd(CONST.TIMING.SEARCH_RENDER);
+    // ASK: Should we also update this name?
+    Timing.end(CONST.TIMING.CHAT_FINDER_RENDER);
+    Performance.markEnd(CONST.TIMING.CHAT_FINDER_RENDER);
```

Comment:
Yea, I think we should update the key, but leave the value of the key as is.
```suggestion
    Timing.end(CONST.TIMING.CHAT_FINDER_RENDER);
    Performance.markEnd(CONST.TIMING.CHAT_FINDER_RENDER);
```